### DISCUSSION TITLE
Getting positional/boxscore like data for individual players prior to 2019

### DISCUSSION THREAD

**Discussion by iWHOswald** [cwendt94/espn-api#167] [ESPN API] [answered]
This API has been very useful for me in pulling down deep stats for my league. I was using the box score class to get matchup and roster specific info for 2019 and 2020, however as your documentation states, 2018 and earlier needs to be done differently. I was attempting to use the scoreboard() function to pull down the actual rosters from these earlier years but have not had success. Is it possible to actually pull down the rosters - specifically starters - earlier than 2019?

**Comment by cwendt94** [comment, marked as answer]
Unfortunately ESPN API doesn't seem to keep weekly matchup roster and stats before the 2019 season. The only roster data you can view are the players that the team ended the season with.

**Reply by iWHOswald** [reply]
Damn yea, even looking at their website it looks like they only display the final rosters as well. That's pretty lame that they discard all that data.

**Reply by cwendt94** [reply]
Yeah I don't know why they didn't keep the old league data. I think it was when they transitioned to the new API in 2019
